### ISSUE TITLE
[New rule] Lint ignore lints that don't ignore any issues

### ISSUE THREAD

**Issue by NickBalnaves** [dart-code-checker/dart-code-metrics#772]
**Please describe what the rule should do:**
Lint when there is an ignore lint comment added to a line that has no issues.

**What category of rule is this? (place an "X" next to just one item)**

[ ] Warns about a potential error (problem)
[X] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about (it will be better if you can provide both good and bad examples):**

BAD:
```dart
// ignore: avoid-dynamic
final String variable = '';
```
```dart
Column(
  children: [
    // ignore: avoid-returning-widgets
    SizedBox(),
  ],
);
```

**Are you willing to submit a pull request to implement this rule?**
No

**Comment by incendial** [comment]
Available in 1.1.0 Teams version release.
